src/components/List/ListItem.scss

```diff
@@ -149,3 +149,29 @@
     }
   }
 }
+
+.date {
+
+ 
+  :first-child {
+    font-size: 19px;
+    font-weight: normal;
+    color: #000;
+    margin-bottom: 2px;
+    text-align: center; 
+  }
+  
+  p {
+      text-align: center; 
+      font-size: 42px;
+      font-weight: normal;
+      color: #000;
+      margin-bottom: 2px;
+  }
+  
+  :last-child {
+      margin-top: 0;
+      font-size: 19px;
```

Comment:
would be great to use em values instead of px for margin and font-size, just to be consistent with other styles. No biggie though, happy to merge